src-vue/src/views/PlayView.vue

```diff
@@ -32,6 +39,11 @@ export default defineComponent({
             <div v-if="northstarVersion !== ''" class="fc_changelog__link" @click="showChangelogPage">
                 (see patch notes)
             </div>
+            <br />
+            <div>
+                {{ playerCount }} Players,
+                {{ serverCount }} Servers
```

Comment:
This does not handle the singular case but honestly I don't think it's worth handling as the `1 player` or `1 server` case is super rare. Looking at player numbers it would only happen when Northstar just got version gated and people are updating. So far we had 3 version gates and the first few players always updated within seconds. ^^

Reply:
I think it's ok not to handle this case as well.